src/connector/postgres.rs

```diff
@@ -0,0 +1,402 @@
+        fn convert(row: &PostgresRow, i: usize) -> QueryResult<ParameterizedValue> {
+            let result = match *row.columns()[i].type_() {
+                PostgresType::BOOL => match row.try_get(i)? {
+                    Some(val) => ParameterizedValue::Boolean(val),
+                    None => ParameterizedValue::Null,
+                },
+                PostgresType::INT2 => match row.try_get(i)? {
+                    Some(val) => {
+                        let val: i16 = val;
+                        ParameterizedValue::Integer(val as i64)
+                    }
+                    None => ParameterizedValue::Null,
+                },
+                PostgresType::INT4 => match row.try_get(i)? {
+                    Some(val) => {
+                        let val: i32 = val;
+                        ParameterizedValue::Integer(val as i64)
+                    }
+                    None => ParameterizedValue::Null,
+                },
+                PostgresType::INT8 => match row.try_get(i)? {
+                    Some(val) => {
+                        let val: i64 = val;
+                        ParameterizedValue::Integer(val)
+                    }
+                    None => ParameterizedValue::Null,
+                },
+                PostgresType::NUMERIC => match row.try_get(i)? {
+                    Some(val) => {
+                        let val: Decimal = val;
+                        let val: f64 = val.to_string().parse().unwrap();
+                        ParameterizedValue::Real(val)
+                    }
+                    None => ParameterizedValue::Null,
+                },
+                PostgresType::FLOAT4 => match row.try_get(i)? {
+                    Some(val) => {
+                        let val: f32 = val;
+                        ParameterizedValue::Real(val as f64)
+                    }
+                    None => ParameterizedValue::Null,
+                },
+                PostgresType::FLOAT8 => match row.try_get(i)? {
+                    Some(val) => {
+                        let val: f64 = val;
+                        ParameterizedValue::Real(val)
+                    }
+                    None => ParameterizedValue::Null,
+                },
+                PostgresType::TIMESTAMP => match row.try_get(i)? {
+                    Some(val) => {
+                        let ts: NaiveDateTime = val;
+                        let dt = DateTime::<Utc>::from_utc(ts, Utc);
+                        ParameterizedValue::DateTime(dt)
+                    }
+                    None => ParameterizedValue::Null,
+                },
+                PostgresType::UUID => match row.try_get(i)? {
+                    Some(val) => {
+                        let val: Uuid = val;
+                        ParameterizedValue::Uuid(val)
+                    }
+                    None => ParameterizedValue::Null,
+                },
+                /*
+                TODO: How to support arrays?
```

Comment:
How should we handle Array types?
A) Add to ParameterizedValue
B) Don't handle at all

Reply:
A.

Reply:
Done.